images/manageiq-base/Dockerfile

```diff
@@ -45,18 +47,43 @@ RUN yum -y install https://rpm.nodesource.com/pub_10.x/el/7/x86_64/nodesource-re
                    lvm2             \
                    net-tools        \
                    nmap-ncat        \
-                   nodejs           \
                    openldap-clients \
                    openscap-scanner \
                    patch            \
                    psmisc           \
-                   postgresql-devel \
                    sqlite-devel     \
                    sysvinit-tools   \
                    which            \
+		   wget 	    \
+		   make		    \
+		   readline-devel   \
+		   zlib-devel	    \
+		   perl-devel	    \
+		   bison	    \
+		   flex		    \
                    &&               \
     yum clean all
 
+RUN wget  https://github.com/postgres/postgres/archive/REL_10_11.tar.gz && \
+    tar -zxvf REL_10_11.tar.gz && cd postgres-REL_10_11/ && ./configure --prefix=/usr && make && make install && \
+    adduser postgres && mkdir -p ${APPLIANCE_PG_DATA} && chown postgres ${APPLIANCE_PG_DATA}
+
+
+# nvm environment variables
+ENV NVM_DIR /usr/local/nvm
+ENV NODE_VERSION 10.17.0
+# install nvm
+# https://github.com/creationix/nvm#install-script
+RUN curl --silent -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.2/install.sh | bash
+# install node and npm
+RUN source $NVM_DIR/nvm.sh \
+    && nvm install $NODE_VERSION \
+    && nvm alias default $NODE_VERSION \
+    && nvm use default
+# add node and npm to path so the commands are available
+ENV NODE_PATH $NVM_DIR/v$NODE_VERSION/lib/node_modules
+ENV PATH $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
```

Comment:
Why are we installing `nvm` and `node` here? I would like to keep this as close to the appliance build as possible where we install the node rpm directly.

Reply:
because this does not work https://github.com/ManageIQ/manageiq-pods/blob/42a6332eeae074c4c2aae8c87846abc80c5f5091/images/manageiq-base/Dockerfile#L48 as https://rpm.nodesource.com/pub_10.x/el/7/x86_64/nodesource-release-el7-1.noarch.rpm is not yet available for Power

Reply:
Created issue here, https://github.com/nodesource/distributions/issues/996